homeassistant/components/climate/fritzhome.py

```diff
@@ -0,0 +1,137 @@
+"""
+Support for AVM Fritz!Box fritzhome thermostate devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/climate.fritzhome/
+"""
+import logging
+
+from homeassistant.components.fritzhome import DOMAIN
```

Comment:
You should not overwrite the climate domain like this. Instead do:
```py
from homeassistant.components.fritzhome import DOMAIN as FRITZHOME_DATA
```
Then use this constant below, where needed.